packages/turf-union/index.ts

```diff
@@ -34,16 +34,17 @@ import { Feature, Polygon, MultiPolygon, Properties } from '@turf/helpers';
  * var addToMap = [poly1, poly2, union];
  */
 function union<P = Properties>(
-    polygon1: Feature<Polygon | MultiPolygon> | Polygon | MultiPolygon,
-    polygon2: Feature<Polygon | MultiPolygon> | Polygon | MultiPolygon,
+    poly1: Feature<Polygon | MultiPolygon> | Polygon | MultiPolygon,
+    poly2: Feature<Polygon | MultiPolygon> | Polygon | MultiPolygon,
     options: {properties?: P} = {}
 ): Feature<Polygon | MultiPolygon, P> {
-    const coords1 = getGeom(polygon1).coordinates;
-    const coords2 = getGeom(polygon2).coordinates;
+    const geom1 = getGeom(poly1);
+    const geom2 = getGeom(poly2);
 
-    const unioned: any = martinez.union(coords1, coords2);
+    // @ts-ignore
```

Comment:
Please try to get rid of the @ts-ignore. See the same comment in turf-intersect/index.js.